report/renderer/performance-category-renderer.js

```diff
@@ -160,6 +161,43 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
     return 'diagnostic';
   }
 
+  /**
+   * @param {LH.ReportResult.AuditRef} audit
+   * @param {LH.ReportResult.AuditRef[]} metricAudits
+   * @return {{overallImpact: number, overallLinearImpact: number}}
+   */
+  overallImpact(audit, metricAudits) {
+    if (audit.result.metricSavings) {
```

Comment:
nit: change this condition to do a `if (!audit.result.metricSavings) return {overallImpact: 0, overallLinearImpact: 0}` to reduce unnecessary indentation.